plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -64,11 +64,13 @@ type MQTTConsumer struct {
 	ConnectionTimeout      config.Duration      `toml:"connection_timeout"`
 	MaxUndeliveredMessages int                  `toml:"max_undelivered_messages"`
 	parser                 parsers.Parser
-	// Legacy metric buffer support; deprecated in v0.10.3
-	MetricBuffer      int
+
+	MetricBuffer      int `toml:"metric_buffer" deprecated:"0.10.3;option is ignored"`
```

Comment:
What do you guys think of adding removal version to 2.0 for this one, as now it throws error as it says it should have been removed in 1.0.0
```suggestion
	MetricBuffer      int `toml:"metric_buffer" deprecated:"0.10.3;2.0.0;option is ignored"`
```

Reply:
Done.
What do you think of the other deprecated options? Shouldn't we add a removal version as well? (since we know they are to be removed in 2.0 according to #9478)

Reply:
The default for the removal date is _next major version_ after deprecation. So you only need to specify it in case you want to diverge from the default, like in this case where the default would have been `1.0.0`.